packages/doc-site/src/components/Dial.md

```diff
@@ -0,0 +1,227 @@
+```jsx
+import { COMPARISON_OPERATOR, StatusIcon, StreamType, DataType } from '@synchro-charts/core';
+<div style={{ width: '100%', height: '300px' }}>
+  <Dial
+    widgetId="test-widget"
+    dataStream={{
+      id: 'car-speed-alarm',
+      name: 'Wind temperature',
+      data: [
+        {
+          x: new Date(2001, 0, 0).getTime(),
+          y: 900,
+        },
+      ],
+      detailedName: 'Medium',
+      resolution: 0,
+      dataType: DataType.NUMBER,
+    }}
+    size={{
+      fontSize: 60,
+      dialThickness: 36,
+      iconSize: 48,
+      labelSize: 32,
+      unitSize: 32,
+    }}  
+    viewport={{ yMin: 0, yMax: 2000, duration: 1000 }}
+  />
+</div>;
+```
+Dial visualizes the latest value from a data stream by emphasizing the property value.For data stream of no unit it will display the percentage by using the two `yMin` and `yMax`.
+
+You can provide `size` to change the font size、dial thickness、icon size、label size、unit size.
+```jsx static
+const size = {
+  fontSize: 60,
+  dialThickness: 36,
+  iconSize: 48,
+  labelSize: 32,
+  unitSize: 32,
+}
+```
+
+```jsx
+import { COMPARISON_OPERATOR, StatusIcon, StreamType, DataType } from '@synchro-charts/core';
+<div style={{ width: '100%', height: '300px' }}>
+  <Dial
+    widgetId="test-widget"
+    dataStream={{
+      id: 'car-speed-alarm',
+      name: 'Wind temperature',
+      data: [
+        {
+          x: new Date(2001, 0, 0).getTime(),
+          y: 911,
+        },
+      ],
+      resolution: 0,
+      dataType: DataType.NUMBER,
+    }}
+    significantDigits={2}
+    size={{
+      fontSize: 60,
+      dialThickness: 36,
+      iconSize: 48,
+      labelSize: 32,
+      unitSize: 32,
+    }}  
+    viewport={{ yMin: 0, yMax: 2000, duration: 1000 }}
+  />
+</div>;
+```
+You can provide `significantDigits` to set the number of significant digits in the primary value.
+```jsx static
+const significantDigits = 2
```

Comment:
not a very useful code example

Reply:
Done --- An example is provided to represent the meaning of `significantDigits`.